### PR TITLE
Adding invoicing policy for review

### PR DESCRIPTION
**Pull request by mkoci-koca**

### CODE REVIEW

File: policies/contractors-invoicing-policy.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+   * The payments CSV batch file will be created and committed for payment no later than **Payment Run Date** from the table below.
+   * If there is an exception, requestor should send it to their manager and to the **finance@openssl.org** mailing list.
+
+5. Manager requests (TO'ing) the treasurer to make the batch payment along with the notice to finance@openssl.org (CC'ing).
+<br><br>
+
+* Additional expenses as approved can be included, subject to another process around making sure they are itemized in the right way for tax (etc). These additional expenses should be submitted via a google form that is created for this purpose. This will send a notification to the finance@openssl.org mailing list too.
+* Manager to batch payments (contractors expenses, invoices) as much as possible, to have a maximum of two potential payment dates a month, and communicate that to contractors.
+
+## Payment Run Schedule Table
```
Comment (paulidale) [inline policies/contractors-invoicing-policy.md:30]: This should be an attachment/appendix/schedule.  It should **not** be part of the policy.

Comment (levitte) [reply policies/contractors-invoicing-policy.md:30]: Agreed.  I'm not even sure why we have that table in public, but fine.  I suggest placing it in the parallel directory `policy-supplemental`, like @TamaraDale did with #24

Comment (mkoci-koca) [reply policies/contractors-invoicing-policy.md:30]: OK, it sounds good

**Comment by levitte** [comment]
#30 supersedes this PR, so this PR could maybe be closed?

**Comment by mkoci-koca** [comment]
OK, closing this PR since #30 supersedes this one.

**Pull request closed without merging**